api/db/changelog/db.changelog-248-add-public-release-number-column.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="eric" id="db.changelog-248-add-public-release-number-column">
+        <addColumn tableName="cdr_version">
+            <column name="public_release_number" type="bigint">
+                <constraints nullable="false"/>
```

Comment:
this might be an issue during 1st time release not sure if that would happen:

1. deploy API - trigger liquibase change, and add this value
2. liquibase add this field with nullable false but all existing rows are null
3. later on, updateCdrConfig insert value into table.
I am worried things would be broken after step 2. To simply, we can just remove this contraints or add later

Reply:
I was thinking similar, but what I realized is that as a numeric field, this defaults to zero instead of null.

This can be demonstrated by removing one of the public release numbers from the local config and then starting the api. If you look in the db, you will see that the cdr that is missing the release number will have a value of zero.